File: omnes.Web/Modules/Parametros/TiposCuentasBancaria/TiposCuentasBancariaForm.cs
using Serenity.ComponentModel;

namespace omnes.Parametros.Forms;

[FormScript("Parametros.TiposCuentasBancaria")]
[BasedOnRow(typeof(TiposCuentasBancariaRow), CheckNames = true)]
public class TiposCuentasBancariaForm
{
    public string Descripcion { get; set; }
    public bool Baja { get; set; }
    public string Observaciones { get; set; }
}

File: omnes.Web/Modules/ServerTypes/Parametros/TiposCuentasBancariaForm.ts
import { StringEditor, BooleanEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface TiposCuentasBancariaForm {
    Descripcion: StringEditor;
    Baja: BooleanEditor;
    Observaciones: StringEditor;
}

export class TiposCuentasBancariaForm extends PrefixedContext {
    static formKey = 'Parametros.TiposCuentasBancaria';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!TiposCuentasBancariaForm.init)  {
            TiposCuentasBancariaForm.init = true;

            var w0 = StringEditor;
            var w1 = BooleanEditor;

            initFormType(TiposCuentasBancariaForm, [
                'Descripcion', w0,
                'Baja', w1,
                'Observaciones', w0
            ]);
        }
    }
}